build.gradle

```diff
@@ -150,10 +148,12 @@ subprojects {
     }
 
     dependencies {
-        api libraries.auto_value_annotation,
```

Comment:
I hate to mention this, but where used and used in android, the annotation jar needs to be set to api. This is one reason we use our own annotations in brave core jar..

https://github.com/google/auto/blob/master/value/userguide/index.md#in-pomxml

Reply:
Do you know why?

Reply:
no I just noticed some android people mentioning this.. maybe @JakeWharton remembers?

Reply:
@adriancole if you hear again about this and you will hear the reason please let me know.

Reply:
well the project is from one of your colleagues @eamonnmcmanus so no need be to be a proxy :p

anyway here's an issue I fished from searching the repo https://github.com/google/auto/issues/620